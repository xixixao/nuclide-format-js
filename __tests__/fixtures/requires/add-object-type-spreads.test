type X = {...Y, a: number};