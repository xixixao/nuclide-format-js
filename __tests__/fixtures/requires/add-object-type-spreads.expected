import Y from 'Y';

type X = {...Y, a: number};
